internal/platform/platform.go

```diff
@@ -29,3 +30,22 @@ func Architecture() string {
 	})
 	return arch
 }
+
+// PossibleCPU returns the set of possible CPUs on the host (which is equal or
+// larger to the number of CPUs currently online). The returned set may be a
+// single CPU number ({0}), or a continuous range of CPU numbers ({0,1,2,3}), or
+// a non-continuous range of CPU numbers ({0,1,2,3,12,13,14,15}).
+func PossibleCPU() []int {
```

Comment:
Not sure if `internal/platform` is the best package for this.
In general we refer to "platform" in the context of an image (OS, architecture, variant, ...) instead of the running system.

Perhaps `internal/system` would be more appropriate? 🤔

Reply:
Pushing back a bit: platform generally means the underlying environment the software is running on, so the number of CPUs falls within that category. 

Having said that, if you feel strongly about it, I can create a separate `internal/system` package, though doing so may raise confusion since we already have a `pkg/system` package (where `PossibleCPU()` would not fit well IMO).

Reply:
Given that there's only a single consumer of this function, the easy way out could be to move these files inside the `oci` packages, and un-export the `possibleCPU()` function

Reply:
I think it fits well in `internal/platform`, as opposed to hiding it within the `oci` package (where it really does not fit, even though it's the only consumer currently).

Reply:
Hi @thaJeztah, @vvoland, any further feedback on this PR? I would like to merge it if possible.

Reply:
> platform generally means the underlying hw environment the software is running on, so the number of CPUs falls within that category.

Right, but in our context (container runtime), platform has a more specific meaning.

How about putting it in a `cpu`/`cpus` package? We could then rename the function name to just `Possible` (so it would make it `cpu.Possible`.

Reply:
> How about putting it in a cpu/cpus package?

We could do that but notice that:

* Package `internal/platform` already has a [NumProcs()](https://github.com/ctalledo/moby/blob/a3fef5debcb0447b84cf87d4cf9494a4cb3287b5/internal/platform/platform_windows.go#L67) function.

* Package `internal/platform` also has an `Architecture()` function which is a property of the CPU.

In other words, there's no clear difference between platform and cpu currently, so I would vote to simplify and keep it all under the platform package.